### PR TITLE
Update appveyor.yml and .travis.yml

### PR DESCRIPTION
**Pull request by thofma**

### CODE REVIEW

**Comment by thofma** [comment]
I will merge it myself.

File: appveyor.yml [hunk 1]
```diff
@@ -1,21 +1,47 @@
 environment:
   matrix:
-  # Releases
-   - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x64/0.5/julia-0.5-latest-win64.exe"
-  # Nightlies
+  - JULIAVERSION: "julialang/bin/winnt/x86/0.5/julia-0.5-latest-win32.exe"
```
Comment (tkelman) [inline appveyor.yml]: this un-does https://github.com/Nemocas/Nemo.jl/pull/173, you should use the new-style urls

File: appveyor.yml [hunk 2]
```diff
@@ -1,21 +1,47 @@
...
+  - JULIAVERSION: "julialang/bin/winnt/x64/0.5/julia-0.5-latest-win64.exe"
+  - JULIAVERSION: "julialang/bin/winnt/x86/0.6/julia-0.6-latest-win32.exe"
+  - JULIAVERSION: "julialang/bin/winnt/x64/0.6/julia-0.6-latest-win64.exe"
+  - JULIAVERSION: "julianightlies/bin/winnt/x86/julia-latest-win32.exe"
+  - JULIAVERSION: "julianightlies/bin/winnt/x64/julia-latest-win64.exe"
+
+matrix:
+  allow_failures:
+    - JULIAVERSION: "julialang/bin/winnt/x86/0.5/julia-0.5-latest-win32.exe"
```
Comment (tkelman) [inline appveyor.yml]: what's wrong with the win32 binaries?

File: appveyor.yml (hunk 1 above)
Comment (thofma) [inline appveyor.yml]: I copied them from https://github.com/JuliaLang/Example.jl yesterday. They were changed right after I did ...

File: appveyor.yml (hunk 2 above)
Comment (thofma) [inline appveyor.yml]: https://github.com/Nemocas/Nemo.jl/issues/171

TL;DR Our Windows 32 binaries are currently not working for Nemo 0.6.

File: appveyor.yml (hunk 1 above)
Comment (tkelman) [inline appveyor.yml]: Example.jl got missed from the first round of PR's to change them everywhere, the script `@staticfloat` used didn't tell him when some PR's failed to open due to the github rate limit

Comment (thofma) [inline appveyor.yml]: I'll update it.
